scrapy/downloadermiddlewares/ajaxcrawl.py

```diff
@@ -89,8 +88,7 @@ def _has_ajaxcrawlable_meta(text):
     if 'content' not in text:
         return False
 
-    text = _script_re.sub(u'', text)
-    text = _noscript_re.sub(u'', text)
-    text = html.remove_comments(html.replace_entities(text))
+    text = html.remove_tags_with_content(text, ('script', 'noscript'))
```

Comment:
Could you please check it with 32K bodies (this is what AjaxCrawlMiddleware passes as `text` argument)? I'm worried about the performance because when AjaxCrawlMiddleware is enabled this function is called for every response; according to https://github.com/scrapy/scrapy/pull/343 existing implementation runs with 2k/sec speed which is not very fast already.